x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/import_rule_modal/index.tsx

```diff
@@ -31,18 +34,36 @@ import {
 import * as i18n from './translations';
 
 interface ImportRuleModalProps {
```

Comment:
Rename the file to match import_data_modal.tsx and also we can rename
```suggestion
interface ImportDataModalProps {
```